src/SamplesApp/UITests.Shared/UITests.Shared.projitems

```diff
@@ -2642,6 +2647,383 @@
     <PRIResource Include="$(MSBuildThisFileDirectory)UITestsStrings\en-US\NamedResources.resw" />
     <PRIResource Include="$(MSBuildThisFileDirectory)UITestsStrings\en-US\Resources.resw" />
   </ItemGroup>
+  <ItemGroup>
+    <Compile Update="C:\Dev\Uno\src\SamplesApp\UITests.Shared\Windows_UI_Xaml_Controls\BorderTests\Border_CornerOverlap.xaml.cs">
```

Comment:
Seems like the latest merge didn't work - there shouldn't be so many changes, and there shouldn't be hard-coded paths.

Reply:
I've filed a bug internally for this, looks like this is a regression in Visual Studio.